naresh/src/main/java/com/lftechnology/phpjava/ems/constants/Constant.java

```diff
@@ -0,0 +1,83 @@
+package com.lftechnology.phpjava.ems.constants;
+
+
+/**
+ * Constant
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 09 2016
+ */
+public class Constant {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1118)